swagger.yaml

```diff
@@ -1,6 +1,6 @@
-swagger: '2.0'
+swagger: "2.0"
```

Comment:
A lot of unrelated changes.
Please move to a different PR if you think it is needed.